doc/manual/source/examples/toolsexample.rst

```diff
@@ -0,0 +1,51 @@
+.. _Tools Example:
+
+:tocdepth: 2
+
+Tools Example
+#############
+
+This example will demonstrate how diffpy.utils allows us to load and manage username and email information.
+Using the tools module, we can efficiently handle username and email configurations stored in ``diffpyconfig.json``.
+
+1) To begin, we may create a configuration file ``diffpyconfig.json`` and add username and email to it. ::
+
+    with ("diffpyconfig.json", "w") as f:
```

Comment:
the instructions shouldn't describe how the code works, but how the user is supposed to used the code.

In this case the "user" is a programmer who wants to capture this information in their app., so the instructions are more like how to implement this behavior in an app they are writing.